Sources/JSONUtilities/DecodingError.swift

```diff
@@ -31,7 +31,7 @@ public struct DecodingError: Error, CustomStringConvertible, CustomDebugStringCo
   /// the value that caused the error
   public let value: Any
 
-  init(dictionary: [AnyHashable: Any], keyPath: StringProtocol, expectedType: Any.Type, value: Any, array: JSONArray? = nil, reason: Reason) {
+  public init(dictionary: [AnyHashable: Any], keyPath: StringProtocol, expectedType: Any.Type, value: Any, array: JSONArray? = nil, reason: Reason) {
```

Comment:
Does this need to become `public`?

Reply:
Since DecodingError is now a Native class in swift I had some problems calling the init, specifying JSONUtilities.DecodingError as public solved the error (I know is public by default).